### DISCUSSION TITLE
Welcome to GITR Discussions!

### DISCUSSION THREAD

**Discussion by dlg0** [ORNL-Fusion/GITR#41] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jguterl** [comment]
Hello, 

I'm Jerome Guterl, staff at General Atomics and working with GITR to model experiments at DIII-D. 
I started working on a workflow to model DIII-D experiments with "hand-written" plasma and impinging particle distribution in collaboration with Tim Younkin, Alyssa Hayes and Ane Lasa.

Details here: https://hackmd.io/@djtyW4n0TgS3u4YCdulk-w/SyA_-ou1u

My suggestions:
- develop a python package to centralize all the pre and post processing for GITR. I started an example [here](https://github.com/jguterl/pyGITR) based on current available python scripts in GITR repo (this is private as it is just temporary. Contact me to get access to it).

- write the documentation with doxygen (GITR source code) and sphinx (pyGITR).    

- Have a workflow  which look like:
>import pyGITR
> SetupGITR() to generate input scripts
> RunGITR() to submit jobs to clusters (Can be wrapped with OMFIT or IPS)

- Ideally, having a direct interface in cpython (i.e. GITR as a cpython library) will tremendously facilitate the use of GITR. See for examples workflows available for LAMMPS.
